EDMarketConnector.py

```diff
@@ -1106,8 +1162,32 @@ def capi_handle_response(self, event=None):  # noqa: C901, CCR001
                 logger.error(msg)
                 raise ValueError(msg)
 
+            if capi_response.capi_data.source_endpoint == companion.session.FRONTIER_CAPI_PATH_FLEETCARRIER:
+                # Fleetcarrier CAPI response
+                # Validation
+                if 'name' not in capi_response.capi_data:
+                    # LANG: No data was returned for the fleetcarrier from the Frontier CAPI
+                    err = self.status['text'] = _('CAPI: No fleetcarrier data returned')
+
+                elif not capi_response.capi_data.get('name', {}).get('callsign'):
```

Comment:
we have deep_get for this

Reply:
`deep_get` is defined in the CLI interface `EDMC.py` and is currently only used there. Should I be importing that into `EDMarketConnector.py`?

Reply:
Leave it for this PR.  If we end up doing that then `deep_get()` needs to end up in `utils/deep_get.py` or somesuch, and that's not a relevant change for this work.